frontend/src/metadata/popups/NewSlicePopup.svelte

```diff
@@ -135,7 +137,10 @@
 				});
 				return s;
 			});
-			selections.update((sels) => ({ slices: [], metadata: sels.metadata }));
```

Comment:
Currently after we create/update the slice, the selection will be cleared and will need to select the modified slice again to see the changes, so I